### ISSUE TITLE
support C++ header files (hh, hpp, hxx)

### ISSUE THREAD

**Issue by scott-vsi** [sbarex/SourceCodeSyntaxHighlight#25]
Can Syntax Highlight be made to support C++ header files (hh hpp hxx)? It looks like highlight supports them.

**Comment by scott-vsi** [comment]
Here is the output of `mdls -name kMDItemContentType -name kMDItemContentTypeTree -name kMDItemKind header.hpp`:

```
kMDItemContentType     = "public.c-plus-plus-header"
kMDItemContentTypeTree = (
    "public.c-plus-plus-header",
    "public.source-code",
    "public.plain-text",
    "public.text",
    "public.data",
    "public.item",
    "public.content"
)
kMDItemKind            = "Visual Studio Code document"
```

**Comment by scott-vsi** [comment]
Perhaps it is just a matter of updating QLExtension/Info.plist and Application/Info.plist

**Comment by sbarex** [comment]
Thanks for the tip, solved on 1.0.b18 release.
